### PR TITLE
Datadog webhook plugin page

### PR DESCRIPTION
**Pull request by chrismcg14**

### CODE REVIEW

File: docs/manual/webhooks/datadog-run-job.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# DataDog (Enterprise)
+
```
Comment (fdevans) [inline docs/manual/webhooks/datadog-run-job.md:4]: Use this code to add the "Enterprise" block that is on the other Enterprise feature pages:
::: enterprise
:::

**Review by fdevans — CHANGES_REQUESTED** [review]

File: docs/manual/webhooks/datadog-run-job.md (hunk 1 above)
Comment (chrismcg14) [reply docs/manual/webhooks/datadog-run-job.md:4]: Included now! @fdevans

**Review by fdevans — APPROVED** [review]
Approved.  We will merge next week during QA.

**Review by fdevans — CHANGES_REQUESTED** [review]
The second D in Datadog is not capitalized

**Comment by fdevans** [comment]
Resolved in another PR

**Pull request closed without merging**